frontend/src/metabase/entities/tables.js

```diff
@@ -74,11 +74,11 @@ const Tables = createEntity({
 
   // ACTION CREATORS
   objectActions: {
-    updateProperty(entityObject, name, value, opts) {
+    updateProperty(entityObject, name, value, skipMerge, opts) {
       return Tables.actions.update(
         entityObject,
         { [name]: value },
-        notify(opts, `Table ${name}`, t`updated`),
+        notify(opts, `Table ${name}`, t`updated`, skipMerge),
```

Comment:
_non-blocking comment_

While we're at it, this string should be translated:

```suggestion
        notify(opts, t`Table ${name}`, t`updated`, skipMerge),
```

And it [looks](https://github.com/metabase/metabase/assets/6830683/b28d124e-fc3f-4021-926f-399011fef151) like "table" should not be capitalized:


```suggestion
        notify(opts, t`table ${name}`, t`updated`, skipMerge),
```